tests/lib/rules/prefer-destructuring.js

```diff
@@ -168,6 +180,7 @@ ruleTester.run("prefer-destructuring", rule, {
         },
         {
             code: "var foobar = object.bar;",
+            output: null,
```

Comment:
It seems like this would also be safe to autofix (though this doesn't need to happen in this PR!)